site/content/extend/plugins/source-available-license.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Source Available License"
+subsection: Plugins (Beta)
+weight: 91
+---
+
+Some plugins authored by Mattermost are licensed under the [Mattermost Source Available License](https://docs.mattermost.com/overview/faq.html#mattermost-source-available-license). This document outlines how to apply the license in various situations.
+
+## How do I apply the license to an enterprise-only plugin?
+
+An enterprise-only plugin is a plugin that requires a valid Mattermost Enterprise E20 license. It is not designed to be used with Team Edition or any other enterprise license.
```

Comment:
```suggestion
An Enterprise-only plugin is a plugin that requires a valid Mattermost Enterprise E20 license. It is not designed to be used with Team Edition or any other Enterprise license.
```

Reply:
Note on this: I'll need to review the other documentation to make sure enterprise/Enterprise is presented correctly so there might be inconsistencies in that right now.